starboard/shared/win32/hardware_decode_target_internal.cc

```diff
@@ -180,6 +180,8 @@ void HardwareDecodeTargetPrivate::InitTextureYUV() {
                                       EGL_TEXTURE_2D,
                                       EGL_TEXTURE_FORMAT,
                                       EGL_TEXTURE_RGBA,
```

Comment:
If those change's are NOT in upstream code, if would recommend adding #ifdef brackets around the added lines. 

`#if defined(STARBOARD)`

or 

`#if defined(USE_COBALT_CUSTOMIZATIONS)`

The purpose is that for the next update of the Angle library re-integration of those change's is easier to manage.

If those are already  in upstream code, then please ignore.

Reply:
It looks like they're not in upstream (assuming you mean the third_party change): https://source.chromium.org/chromium/chromium/src/+/main:third_party/angle/src/libANGLE/validationEGL.cpp;l=3022;drc=046327343595c94f82d3075451c968cbe5ee8320

Once you add the def guards this should be good to submit

Reply:
It looks like our fix to the main Angle repository has succesfully passed all required steps, but wasn't merged for more, than a month:
https://chromium-review.googlesource.com/c/angle/angle/+/5646493
Could you ask them for merging, because we want finish our Angle tasks while we have hands on them?